src/CompositeRule.php

```diff
@@ -36,7 +36,7 @@ final class CompositeRule implements RuleInterface
 
     /**
      * @param string $operator Operator to be used. Could be `CompositeRule::AND` or `CompositeRule::OR`.
-     * @param RuleInterface[] $rules Array of rule instances.
+     * @param array|RuleInterface[] $rules Array of rule instances.
```

Comment:
`$rules` can only be array of `RuleInterface`. Why added array of mixed values?

Reply:
Fixed.